packages/loot-core/src/server/api.ts

```
@@ -768,6 +776,166 @@
+          conditionsUpdated = true;
+        } else {
+          throw APIError('There is already a schedule with this name');
+        }
+        break;
+      }
+      case 'next_date':
+      case 'completed': {
+        throw APIError(`Field '${typedKey}' is system-managed and not user-editable.`);
+        break;
+      }
+      case 'posts_transaction': {
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)